src/pkgcheck/checks/codingstyle.py

```diff
@@ -903,3 +903,102 @@ def feed(self, pkg):
                         yield RedundantDodir(
                             cmd.group('cmd'), line=dodir.group('call'),
                             lineno=lineno - 1, pkg=pkg)
+
+
+class EbuildUnquotedVariable(results.LineResult, results.Style):
+    """Variable is unquoted, but should be quoted"""
+
+    def __init__(self, variable, **kwargs):
+        super().__init__(**kwargs)
+        self.variable = variable
+
+    @property
+    def desc(self):
+        return f"{self.lineno}: Variable {self.variable} should be quoted"
+
+
+class EclassUnquotedVariable(results.EclassResult, results.Style):
```

Comment:
It should be possible to collapse both the ebuild and eclass results using the style done for results in the header module. For example see the InvalidCopyright generic result and the related EbuildInvalidCopyright and EclassInvalidCopyright noting the class attr `_name` set in the parent class.

Reply:
I followed the approach from InvalidCopyright, and collapsed the output to a line per variable. Thanks for the suggestion!